monai/bundle/utils.py

```diff
@@ -104,22 +104,25 @@
 DEFAULT_MLFLOW_SETTINGS = {
     "handlers_id": DEFAULT_HANDLERS_ID,
     "configs": {
+        "tracking_uri": "$@output_dir + '/mlflow'",
```

Comment:
Hi @Nic-Ma ,

      I'd like to suggest that changes the `/mlflow` to `/mlruns`. As the [link](https://mlflow.org/docs/latest/tracking.html#tracking-ui) shows, mlflow ui will only check the `mlruns` folder. If another name is given, it will not show the recorded result when running the `mlflow ui` command.

Thanks,
Bin

Reply:
OK, I will update.
And BTW, you can run `mlflow ui --backend-store-uri <PATH>` to use any folder name.

Thanks.